embedded-hal-bus/Cargo.toml

```diff
@@ -15,9 +15,13 @@ repository = "https://github.com/rust-embedded/embedded-hal"
 version = "0.2.0"
 
 [features]
-std = []
+std = ["alloc"]
 async = ["dep:embedded-hal-async"]
 defmt-03 = ["dep:defmt-03", "embedded-hal/defmt-03", "embedded-hal-async?/defmt-03"]
+# Enables additional utilities requiring a global allocator.
+alloc = []
+# TODO: remove this
+default = ["alloc"]
```

Comment:
```suggestion
```

Reply:
Oh great I managed to forget to remove it :')